.github/workflows/build-yad.yml

```diff
@@ -0,0 +1,78 @@
+name: Build and release yad
+
+on:
+  workflow_dispatch:
+
+env:
+  YAD_VERSION: "13.0"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-22.04
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install required packages
+      run: sudo apt-get install autoconf automake intltool libgtk-3-dev libglib2.0-dev libwebkit2gtk-4.0-dev
+
+    - name: Retrieve Yad source code
+      run: |
+          git clone https://github.com/v1cont/yad/ yad-dialog-code
+          cd yad-dialog-code
+          git checkout v$YAD_VERSION
+
+    - name: Retrieve linux deploy
+      run: |
+          wget https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage
+          chmod +x ./linuxdeploy-x86_64.AppImage
+          ./linuxdeploy-x86_64.AppImage --appimage-extract
+
+    - name: Build Yad
+      working-directory: yad-dialog-code
+      run: |
+          autoreconf -ivf
+          intltoolize
+          ./configure prefix=$(pwd)/AppDir --enable-standalone
+          make
+          make install
+
+    - name: Prepare .desktop file
+      working-directory: yad-dialog-code
+      run: |
```

Comment:
Maybe I'm being silly, but do we need to create a `.desktop` file? If this is going to be done on the runner and not bundled with a release.

Maybe it's required for some other purpose, I do see it in my `easy-build` local files, and on the `easy-builder` script, so let me know if this is needed. Maybe there's some quirk where the AppImage expects it. Since Yad isn't really going to get "integrated" into the system, although AppImageLauncher would normally take care of that but maybe it pulls from this `.desktop` file.

It's not gonna make a huge difference either way, I would say it takes fractions of a second to make this file, this is really just curiosity on my part :-) If you are unsure like me and want to err on the side of caution and keep it in, it's not going to bother me.

Reply:
After some digging it looks mandatory because it appears to make linuxdeploy know which file needs to be executed when running the appimage. See https://docs.appimage.org/reference/appdir.html#general-description and https://github.com/linuxdeploy/linuxdeploy/issues/138

Reply:
Thanks for taking the time to dig into it! I'll mark this as resolved.